Tools/TestWebKitAPI/Tests/WebKitGLib/TestWebKitFindController.cpp

```diff
@@ -20,8 +20,19 @@
 #include "config.h"
 
 #include "LoadTrackingTest.h"
+#include "WebKitImagePrivate.h"
 #include <wtf/glib/GRefPtr.h>
 
+#if PLATFORM(GTK)
+#if USE(GTK4)
+#define SNAPSHOT_TYPE GdkTexture
+#else
+#define SNAPSHOT_TYPE cairo_surface_t
+#endif
+#else
```

Comment:
...and the same for the tests:

```suggestion
#elif ENABLE(2022_GLIB_API)
```